src/Server/Repositories/ClaraPayloadsApi.cs

```diff
@@ -39,9 +38,11 @@ public class ClaraPayloadsApi : IPayloads
         private readonly IFileSystem _fileSystem;
         private readonly IPayloadsClient _payloadsClient;
 
-        public ClaraPayloadsApi(IOptions<DicomAdapterConfiguration> dicomAdapterConfiguration, ILogger<ClaraPayloadsApi> logger) : this(
-            InitializePayloadsClient(dicomAdapterConfiguration),
-            logger,
+        public ClaraPayloadsApi(
```

Comment:
The client is actually provided by Platform in `Nvidia.Clara.Platform.IPayloadsClient`, to avoid confusion, this is placed inside the `Repositories` dir as an API.